wait/member.go

```diff
@@ -570,3 +571,30 @@ func (a *MemberAwaitility) WaitForMemberStatus(criteria ...MemberStatusWaitCrite
 	})
 	return memberStatus, err
 }
+
+// WaitForMetricsService waits until there's a service called `host-operator-metrics` in the host
```

Comment:
I guess this should be `member-operator-metrics` in the comment?

Reply:
ah yes, thanks! 
Fixed in 2a79260